src/components/Sing/ScoreSequencer.vue

```diff
@@ -744,13 +744,13 @@ const onMouseDown = (event: MouseEvent) => {
   if (!isSelfEventTarget(event)) {
     return;
   }
-  if (event.shiftKey) {
-    isQuadSelecting.value = true;
-    quadSelectStartX.value = cursorX;
-    quadSelectStartY.value = cursorY;
-    return;
-  }
   if (event.button === 0) {
+    if (event.shiftKey) {
+      isRectSelecting.value = true;
+      rectSelectStartX.value = cursorX;
+      rectSelectStartY.value = cursorY;
+      return;
+    }
     startPreview(event, "ADD");
     mouseDownNoteId = undefined;
```

Comment:
`mouseDownNoteId = undefined;`も実行する必要があるかもです！
（ダブルクリック周り整理できてなくてすみません…）
```suggestion
    if (event.shiftKey) {
      isRectSelecting.value = true;
      rectSelectStartX.value = cursorX;
      rectSelectStartY.value = cursorY;
    } else {
      startPreview(event, "ADD");
    }
    mouseDownNoteId = undefined;
```